mopidy/internal/network.py

```diff
@@ -138,9 +139,15 @@ def stop(self):
         if unix_socket_path is not None:
             os.unlink(unix_socket_path)
 
-    def register_server_socket(self, fileno):
+    def register_server_socket(self, sock):
+        # GLib wants socket for win32, file-num for others
+        if sys.platform == 'win32':
+            chan = sock
+        else:
+            chan = sock.fileno()
         return GLib.io_add_watch(
-            fileno,
+            chan,
+            1,  # priority?
```

Comment:
Yes, from https://developer.gnome.org/glib/stable/glib-IO-Channels.html#g-io-add-watch-full it looks like this is probably the priority. Do we have to pass it?

Reply:
If the priority is omitted, GLib emits a warning about it that pytest interprets as test failure.  Online docs at https://developer.gnome.org/pygobject/stable/glib-functions.html#function-glib--io-add-watch name that second parameter 'condition', and describe it as a bit mask.  The value should probably be one of the named values, perhaps GLib.IO_IN (or all of them |ed).  I'll reexamine the source code this afternoon, and find a basis for a specific value there.